public/index.html

```diff
@@ -14,6 +14,9 @@
     <meta property="og:image" content="https://wakamaifondue.com/wf-screenshot-square.jpg" />
     <link rel="icon" type="image/svg+xml" href="<%= BASE_URL %>favicon.svg">
     <title>Wakamai Fondue, the tool that answers the question “what can my font do?”</title>
+    <script defer
+        src="//wakamaifondue.trackless.app/analytics.js"
```

Comment:
Just checking: this doesn't use cookies?

Reply:
It doesn't! But see also my [other comment](https://github.com/Wakamai-Fondue/wakamai-fondue-site/pull/120#discussion_r580889431). :)